src/Microsoft.ML.Core/Data/ModelLoadContext.cs

```diff
@@ -40,6 +40,11 @@ internal sealed partial class ModelLoadContext : IDisposable
         /// </summary>
         public readonly string[] Strings;
 
+        /// <summary>
+        /// The name of the assembly that the loader lives in. This property is used for wrapping old ML.StandardLearners assembly.
+        /// </summary>
+        public readonly string ForwardedLoaderAssemblyName => LoaderAssemblyName == "Microsoft.ML.StandardLearners" ? "Microsoft.ML.StandardTrainers" : LoaderAssemblyName;
```

Comment:
>readonly [](start = 15, length = 8)

Note that unlike the other things here which are fields, what you are writing here is a property, and the `readonly` modifier will not work for it.

Reply:
I would recommend removing this, syncing to master and updating the existing `[ForwardedLoaderAssemblyName](https://github.com/dotnet/machinelearning/blob/df2f3fb49d4299925325921ee31c2cdff78ba3b5/src/Microsoft.ML.Core/Data/ModelLoading.cs#L27)`, StandardLearners isn't the only assembly that requires forwarding.